cou/steps/__init__.py

```diff
@@ -27,6 +27,7 @@
 
 logger = logging.getLogger(__name__)
 DEPENDENCY_DESCRIPTION_PREFIX = "├── "
+TAB = "\t"
```

Comment:
Unnecessary constant

Reply:
If you want to can rather create function to join units with `\t- {unit.name}`, instead of using `units_repr = "\n".join([f"{TAB}- '{unit.name}'" for unit in units])`.